### DISCUSSION TITLE
Apple Silicon

### DISCUSSION THREAD

**Discussion by Irek7** [sc0ty/subsync#171] [General]
Hi.

Thanks for your great work.

Can you prepare native version for Apple Silicon?

br
Irek

**Comment by lukecrownet** [comment]
I'm also on Apple silicon (and macOS 14.2.1) -- and trying to open the app (or reinstalling via the DMG) results in "this app is damaged" errors along with recommendations by the OS to eject the DMG and delete the app... Any chance of a native/working version of this app for modern Apple computers? If not -- many thanks for maintaining https://subsync.online/en/online.html -- it works amazingly well, even if very tricky subtitles and frame-rate drifts, etc. Very cool.
